kernel/src/tty/reader.rs

```diff
@@ -57,6 +57,7 @@ impl<'a> Reader<'a> {
             "nil" => MalType::Nil,
             "true" => MalType::Bool(true),
             "false" => MalType::Bool(false),
+            str if str.starts_with('\"') => MalType::String(str[1..str.len() - 1].to_string()),
```

Comment:
nit: `str` is quite hard to read with syntax highlighting as a variable name

Reply:
Fair point